aws_lambda_builders/workflow.py

```diff
@@ -22,6 +25,40 @@
 Capability = namedtuple('Capability', ["language", "dependency_manager", "application_framework"])
 
 
+# TODO: Move sanitize out to its own class.
+def sanitize(func):
+    """
+    sanitize the executable path of the runtime specified by validating it.
+    :param func: Workflow's run method is sanitized
+    """
+
+    @functools.wraps(func)
+    def wrapper(self, *args, **kwargs):
+        valid_paths = []
+        # NOTE: we need to access binaries to get paths and resolvers, before validating.
+        self.get_binaries()
```

Comment:
We call `self.get_binaries()` but don't use the output at all?

Reply:
good catch, it sets self.binaries, so I just used that directly.